src/crowdin_client.cpp

```diff
@@ -200,8 +200,8 @@ dispatch::future<CrowdinClient::UserInfo> CrowdinClient::GetUserInfo()
             catch(...)
             {
                 std::string firstName, lastName;
-                try { firstName = d.at("firstName"); } catch(...) {}
-                try { lastName = d.at("lastName"); } catch(...) {}
+                try { firstName = d.at("firstName").get<std::string>(); } catch(...) {}
+                try { lastName = d.at("lastName").get<std::string>(); } catch(...) {}
```

Comment:
> The commit's message is misleading as this is not about wxStandardPaths (and such fixups are best handled by fixing up the bad topical commit, but never mind)

Yep, I see there some changes unrelated to this commit intention at all leaked in even after arranging commits to make them more clean and clear in this PR. e.g. [like this one](https://github.com/vslavik/poedit/pull/633/files/e06429f3e0217d757a81ed0e19071e82e35570e7..98ab380263e281e8c71f281208bd22fa0140990f#diff-a1865a3025926eb82afbd7f4d9f13518). Also description remained inaccurate (wxStandardPaths() fix was made in some another commit or just was not needed any more). Though trying to avoid rebases in this final PR already unless really would make more convenient for you to continue review/discussion by PR, since after rebase GitHub usually loses references on places in source code in PR review saying that "can not found them due of most probable rebased" ... unless I just miss some trick to go around of this. So I hope you still are able to catch/follow purpose of changes in this commit or otherwise I will be happy to clarify everything you will ask/comment. 

> I think what you meant by "Buildable on Windows" was "Buildable with Visual Studio 16.5" and the correct fix for it landed on master in 03ba307, rendering most of the changes in the commit unneeded.

Addressed 2c5a6b710e97936fc251d114b443ce9eac00b79d

Reply:
But still c65cfeff45c01ad311f50c9642fe23463c2e1042